frontend/package.json

```diff
@@ -32,8 +32,7 @@
     "server"
   ],
   "engines": {
-    "node": ">=18.14.2",
-    "npm": ">=9.5.0"
+    "node": "^20"
```

Comment:
If we are updating the node version we should also update the npm version rather than just deleting it
e.g. `npm: >=10.2.0`

Reply:
since npm comes with node it makes no much sense to have both but I'll add back :+1: